src/pages/MetricDetail.svelte

```diff
@@ -138,6 +138,8 @@
         <tr>
           <td><code>{keyName}</code></td>
           <td>
+            {#if definition.type}<td>Type: <code>{definition.type}</code></td
```

Comment:
Hmm, I'm not sure why this is nested inside another `<td/>`?

I think we should always have a column for type and say it's `undefined` (or something similar) when it isn't present.

Reply:
it's "string" when not defined